ql/src/test/results/clientpositive/llap/udaf_number_format.q.out

```diff
@@ -92,4 +92,4 @@ FROM src
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@src
 #### A masked pattern was here ####
-0.0	0.0	NULL	NULL
+0.0	NULL	NULL	NULL
```

Comment:
I believe this diff is expected based on the discussion.
https://github.com/apache/hive/blob/e78e27bc593c20a312c76b7ba9c06b96176bc647/ql/src/test/queries/clientpositive/udaf_number_format.q